src/VisualStudio/Core/Def/Storage/FileDownloader.cs

```diff
@@ -19,7 +19,7 @@ public IFileDownloader CreateClient(string hostId, string serverPath, int pollin
         {
             // BaseUrl provided by the VS RemoteControl client team.  This is URL we are supposed
             // to use to publish and access data from.
-            const string BaseUrl = "https://az700632.vo.msecnd.net/pub";
+            const string BaseUrl = "https://aka.ms/vssettings/pub";
```

Comment:
i'd like to debug through this to make sure this works :)  i can do so tomorrow.